python/setup.py

```diff
@@ -64,7 +64,7 @@ def get_cuda_version_from_header(cuda_include_dir):
 
 cwd = os.getcwd()
 preprocess_files = ["gpu.pxd"]
-supported_cuda_versions = {"10.1", "10.2", "11.0"}
+supported_cuda_versions = {"10.1", "10.2", "11.0", "11.2"}
```

Comment:
Are we thinking about 11.1 as well or are we skipping that?

Reply:
Ah good point -- shall I go ahead and add it? I just realized we don't have CI support 11.1 or 11.2 just yet.

Reply:
@kkraus14 wdyt?

Reply:
May as well add it for now. We're planning on trying to use CUDA Enhanced Compatibility in the future which means we'll generally want to build with the latest 11.x, but it's undecided if we intend to continue supporting compiling with older CUDA toolkit versions or just running on them when building with the latest 11.x

Reply:
Added 11.1 support as well -- I haven't tested it locally though